File: API/Client/WarehouseDTO.js
goog.provide('API.Client.WarehouseDTO');

/**
 * Информация о складе.
 * @record
 */
API.Client.WarehouseDTO = function() {}

/**
 * Идентификатор склада.
 * @type {!number}
 * @export
 */
API.Client.WarehouseDTO.prototype.id;

/**
 * Название склада.
 * @type {!string}
 * @export
 */
API.Client.WarehouseDTO.prototype.name;

/**
 * Идентификатор кампании в API и идентификатор магазина.
 * @type {!number}
 * @export
 */
API.Client.WarehouseDTO.prototype.campaignId;

/**
 * Возможна ли доставка по модели Экспресс.
 * @type {!boolean}
 * @export
 */
API.Client.WarehouseDTO.prototype.express;

/**
 * @type {!API.Client.WarehouseAddressDTO}
 * @export
 */
API.Client.WarehouseDTO.prototype.address;

File: src/gatling/scala/org/openapitools/client/model/WarehouseDTO.scala

package org.openapitools.client.model


case class WarehouseDTO (
    /* Идентификатор склада. */
    _id: Long,
    /* Название склада. */
    _name: String,
    /* Идентификатор кампании в API и идентификатор магазина. */
    _campaignId: Long,
    /* Возможна ли доставка по модели Экспресс. */
    _express: Boolean,
    _address: Option[WarehouseAddressDTO]
)
object WarehouseDTO {
    def toStringBody(var_id: Object, var_name: Object, var_campaignId: Object, var_express: Object, var_address: Object) =
        s"""
        | {
        | "id":$var_id,"name":$var_name,"campaignId":$var_campaignId,"express":$var_express,"address":$var_address
        | }
        """.stripMargin
}
